soc/openhwgroup/cva6/Kconfig.defconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_OPENHWGROUP_CVA6
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 25000000
```

Comment:
should come from dts property

Reply:
Done (from `timebase-frequency` property in cpus)